tests/phpunit/tests/admin/wpAutomaticUpdater.php

```diff
@@ -0,0 +1,572 @@
+<?php
+
+/**
+ * @group upgrade
+ *
+ * @covers WP_Automatic_Updater
+ */
+class Tests_Admin_WpAutomaticUpdater extends WP_UnitTestCase {
+	/**
+	 * An instance of WP_Automatic_Updater.
+	 *
+	 * @var WP_Automatic_Updater
+	 */
+	private static $updater;
+
+	/**
+	 * WP_Automatic_Updater::send_plugin_theme_email
+	 * made accessible.
+	 *
+	 * @var ReflectionMethod
+	 */
+	private static $send_plugin_theme_email;
+
+	/**
+	 * Sets up shared fixtures.
+	 */
+	public static function set_up_before_class() {
```

Comment:
Needs a call to `parent::set_up_before_class();`

The docs are a little confusing as to whether to use this or `wpSetUpBeforeClass` (which doesn't require a `parent::` call).

The [section on slow fixtures suggest the `wp` prefixed function](https://make.wordpress.org/core/handbook/testing/automated-testing/writing-phpunit-tests/#slow-fixtures),  but the [section on shared fixtures suggests the snake case function](https://make.wordpress.org/core/handbook/testing/automated-testing/writing-phpunit-tests/#shared-fixtures).

Reply:
`set_up_before_class()` and `tear_down_after_class()` call their `wp` equivalents. [ref](https://github.com/WordPress/wordpress-develop/blob/trunk/tests/phpunit/includes/abstract-testcase.php#L63).

I _think_ the "Slow Fixtures" section is more about using the "before class" option rather than "before test" methods.

Reply:
Also yes, needs a call to the parent method. My bad.

Reply:
Switched to `wpSetUpBeforeClass()` in [2722e14](https://github.com/WordPress/wordpress-develop/pull/3100/commits/2722e147d397bdeaa9efce7cd143d8fce005a2d4).